src/MixAndRender.cpp

```diff
@@ -131,6 +131,7 @@ void MixAndRender(TrackList *tracks, WaveTrackFactory *trackFactory,
    using namespace BasicUI;
    auto updateResult = ProgressResult::Success;
    {
+      auto effectiveFormat = mixer.EffectiveFormat();
```

Comment:
As I wrote to you in more detail by email, this call returns int16 when simply trying to mix two 32-bit tracks created on the spot - I might be wrong here, but I would expect it to return floatSamples instead

Reply:
Congratulations, you found a real mistake of mine.

Reply:
I re-tested this, and confirm it is now fixed.